ActivityTracker/background.js

```diff
@@ -28,6 +33,61 @@ function zeroPrefixedNum(num, width) {
     return prefix + numStr;
 }
 
+function getActiveTab() {
+    return new Promise((resolve, reject) => {
+        chrome.tabs.query({active: true, currentWindow: true}, function (tabs) {
+            const activeTab = tabs[0];
+            if (!activeTab) {
+                if(reject) {
```

Comment:
do we really need this check? both `resolve` and `reject` should be always present, pls double check this